frontend/src/components/common/presentational/profilePageHeader/ProfilePageHeader.tsx

```diff
@@ -45,7 +52,7 @@ function ProfilePageHeader() {
                   marginRight: "10.5px",
                 }}
               />
-              <p>Number</p>
+              <p>{userData?.number ? `{userData?.number}` : "Number"}</p>
```

Comment:
Don't add this ternary op. Just show number value. If it is empty, let it be empty.

Reply:
Sure